### PR TITLE
Rename agora.common.Data to agora.common.Types

### PR DESCRIPTION
**Pull request by Geod24**
The previous name wasn't very adequate and quite generic.
The new name is still pretty generic, but closer to it's intent.

### CODE REVIEW

File: source/agora/cli/SendTxProcess.d [hunk 1]
```diff
@@ -16,7 +16,7 @@ module agora.cli.SendTxProcess;
 import agora.cli.CLIResult;
 import agora.common.Amount;
 import agora.common.crypto.Key;
-import agora.common.Data;
+import agora.common.Types;
```
Comment (AndrejMitrovic) [inline source/agora/cli/SendTxProcess.d]: Reorder after Hash.

File: source/agora/common/Metadata.d [hunk 2]
```diff
@@ -13,7 +13,7 @@
 
 module agora.common.Metadata;
 
-import agora.common.Data;
+import agora.common.Types;
```
Comment (AndrejMitrovic) [inline source/agora/common/Metadata.d]: Reorder after Set.

**Comment by AndrejMitrovic** [comment]
Yeah just reorder and I'll merge.

**Comment by Geod24** [comment]
Fixed

**Review by AndrejMitrovic — APPROVED** [review]

**Pull request merged**